tests/ha/ha_cluster_join.pm

```diff
@@ -13,10 +13,26 @@
 use base 'opensusebasetest';
 use strict;
 use warnings;
-use testapi;
+use testapi qw(is_serial_terminal :DEFAULT);
 use lockapi;
 use hacluster;
-use utils 'zypper_call';
+use utils qw(zypper_call);
+
+sub wait_for_password_prompt {
+    my %args = @_;
+    $args{timeout} //= $default_timeout;
+    $args{failok}  //= 0;
+    die 'wait_for_password_prompt() expects a needle tag or ref in the "needle => $tag" arg' unless $args{needle};
```

Comment:
`wait_for_password_prompt()` could be more dynamic and replaced by `(caller(0))[3]` (it is already used in other openQA lib). But I honestly don't know if it's really "readable" :D

Reply:
Will change it. Thanks!